_pages/fsa/inbox.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Inbox
+---
+
+## Overview
+The Inbox feature enables your customers to see all notifications in one place, that is, on the Inbox page of the My Account area. All messages are grouped according to their content, sorted, and then displayed in different tabs.
+
+More details regarding Inbox feature from Financial Services Accelerator could be found
+ [here](https://help.sap.com/viewer/4c33bf189ab9409e84e589295c36d96e/1905/en-US/abe842cac00a4f34a756cd720d4c2288.html).
+What's new is that now messages in inbox can have attachments. Customer can see icon as indicator of some attachments, also by clicking on attachment link customer will be redirected to the attachment url. 
```

Comment:
What's new is that now messages in inbox can have attachments. Maybe to change that sentence with: In addition to that, messages in Inbox can have attachments.